OpenRA.Mods.Common/Traits/Cargo.cs

```diff
@@ -47,9 +47,21 @@ public class CargoInfo : ITraitInfo, Requires<IOccupySpaceInfo>
 		[Desc("Voice to play when ordered to unload the passengers.")]
 		public readonly string UnloadVoice = "Action";
 
+		[Desc("Radius within which the actor will search for a suitable unloading spot when the ordered location is blocked")]
+		public readonly int UnloadSearchRadius = 5;
+
 		[Desc("Which direction the passenger will face (relative to the transport) when unloading.")]
 		public readonly int PassengerFacing = 128;
 
+		[Desc("How long should this actor wait after loading")]
+		public readonly int WaitAfterLoad = 8;
```

Comment:
Trait naming conventions are to use `*Delay` for delays, and to mention that they are in ticks in the description.

Reply:
These could therefore become
```csharp
[Desc("Delay (in ticks) before continuing after loading a passenger.")]
public readonly int AfterLoadDelay = 8;

[Desc("Delay (in ticks) before unloading the first passenger.")]
public readonly int BeforeUnloadDelay = 8;

[Desc("Delay (in ticks) before continuing after unloading a passenger.")]
public readonly int AfterUnloadDelay = 25;
```

Reply:
Done.